templates/pricing/pro.html

```diff
@@ -12,7 +12,7 @@
 
 <section id="ua-infra" class="p-strip--suru-topped is-bordered">
   <div class="u-fixed-width">
-    <h1>Ubuntu Pro:  security, compliance, and support</h1>
+    <h1>Ubuntu Pro: security, compliance</h1>
```

Comment:
In the copy docs this is `Ubuntu Pro: security, compliance, and support` just checking if you are aware of it?

Reply:
Your right of course. Updated :+1: